src/helpers/selectors.ts

```diff
@@ -45,22 +46,30 @@ export const getSettingsSwitch = (
     { visible: true }
   );
 
-export const getErrorMessage = async (page: Page): Promise<string | false> => {
-  const options: Parameters<Page["waitForSelector"]>[1] = { timeout: 1000 };
+export const getErrorMessage = async (
+  page: DappeteerPage
+): Promise<string | false> => {
+  const options: Parameters<DappeteerPage["waitForSelector"]>[1] = {
+    timeout: 1000,
+  };
 
   // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
-  const errorElement: ElementHandle<HTMLElement> | null = await Promise.race([
-    page.waitForSelector(`span.error`, options),
-    page.waitForSelector(`.typography--color-error-1`, options),
-    page.waitForSelector(`.typography--color-error-default`, options),
-  ]).catch(() => null);
+  const errorElement: DappeteerElementHandle<Serializable> | null =
+    await Promise.race([
+      page.waitForSelector(`span.error`, options),
```

Comment:
As a suggestion, we can use testing-library to have similar selectors regardless of the framework or platform we are using. I find it annoying to learn all the different ways of searching for the element as implementations are different. So, why not use the library which is widely used across developers to have a similar API for querying the UI elements? 

https://testing-library.com/docs/pptr-testing-library/intro
https://github.com/testing-library/playwright-testing-library

Reply:
Seems good, could you open issue with your suggested approach?